docs/sources/tempo/configuration/grafana-agent/_index.md

```diff
@@ -23,13 +23,25 @@ On top of receiving and exporting traces, the Grafana Agent contains many
 features that make your distributed tracing system more robust, and
 leverages all the data that is processed in the pipeline.
 
+## Agent modes
+
+Grafana Agent is available in three different variants:
+
+* [Static mode](/docs/agent/latest/static): The original Grafana Agent.
+* [Static mode Kubernetes operator](/docs/agent/latest/operator): The Kubernetes operator for Static mode.
```

Comment:
my impression is that the agent team is looking to deprecate the k8s operator. let's remove this reference.

Reply:
Done.